projects/relay_eth/releases.json

```diff
@@ -1,43 +1,60 @@
+                                    "geth_checksum": "1d856a8e29bf26ed0f687b5108d11091", 
+                                    "relay": "http://public.artifacts.marlin.pro/projects/relay_eth/1.0.0/eth_relay-1.0.0-linux-amd64", 
+                                    "relay_checksum": "c08f2002bbd686710b7117c2d64cbc1a"
+                                }, 
+                                "runner": "linux-amd64.supervisor.runner01"
                             }
-                        }
+                        }, 
+                        "description": "1.0.0 initial release", 
+                        "time": "02 Jan 21 18:00 +0530"
                     }
-                },
+                }, 
                 "1": {
                     "0": {
-                        "time": "08 Jan 21 14:00 +0530",
-                        "description": "1.0.1 bug fix",
                         "bundles": {
                             "linux-amd64.supervisor": {
-                                "runner": "linux-amd64.supervisor.runner01",
                                 "data": {
-                                    "relay": "http://public.artifacts.marlin.pro/projects/relay_eth/1.0.1/relay_eth-linux_amd64",
-                                    "relay_checksum": "6ea506fcfd829409d04da048ba8599f4",
-                                    "geth": "http://public.artifacts.marlin.pro/projects/relay_eth/1.0.1/geth-linux_amd64",
-                                    "geth_checksum": "f31591d47a91b530f99109774eb662f2"
-                                }
+                                    "geth": "http://public.artifacts.marlin.pro/projects/relay_eth/1.0.1/geth-linux_amd64", 
+                                    "geth_checksum": "f31591d47a91b530f99109774eb662f2", 
+                                    "relay": "http://public.artifacts.marlin.pro/projects/relay_eth/1.0.1/relay_eth-linux_amd64", 
+                                    "relay_checksum": "6ea506fcfd829409d04da048ba8599f4"
+                                }, 
+                                "runner": "linux-amd64.supervisor.runner01"
                             }
-                        }
+                        }, 
+                        "description": "1.0.1 bug fix", 
+                        "time": "08 Jan 21 14:00 +0530"
+                    }
+                }, 
+                "2": {
+                    "0": {
+                        "bundles": {
+                            "linux-amd64.supervisor": {
+                                "data": {
+                                    "geth": "https://public.artifacts.marlin.pro/projects/relay_eth/1.0.2/geth-linux_amd64", 
+                                    "geth_checksum": "89f062311ad82f8e74dee1aebfb77caa", 
+                                    "relay": "https://public.artifacts.marlin.pro/projects/relay_eth/1.0.1_bkp/relay_eth-linux_amd64", 
```

Comment:
1. No `https`
2. Shift 1.0.1 artifact to 1.0.2 itself or create 1.0.3 if facing cache invalidation problem